pkg/strategy/strategy.go

```diff
@@ -9,12 +9,13 @@ import (
 	"k8s.io/api/extensions/v1beta1"
 	rbac "k8s.io/api/rbac/v1"
 
+	jv1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
```

Comment:
We typically use `v1` for Jaeger, and `corev1` for `k8s.io/api/core/v1`

Reply:
Makes a lot of sense!

I saw that `v1` was already in use and invented a different alias for Jaeger. Didn't think about the possibility of renaming the existing one :). Will do it now.